packages/gatsby/src/query/graphql-runner.js

```diff
@@ -1,4 +1,5 @@
-const { graphql } = require(`graphql`)
+const { parse, validate, execute } = require(`graphql`)
```

Comment:
One possible change here would be to manage nodeModel cache state here too. Currently we recreate GraphQLRunner from develop/build queue, it's also possible to handle all the lifecycle here in the runner. It might be a good separate PR, though.

Reply:
Yeah, I tried other refactoring to enable the same result. Will publish it as a separate PR but it has its own cons.